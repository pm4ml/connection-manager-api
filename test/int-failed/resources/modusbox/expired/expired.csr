-----BEGIN CERTIFICATE REQUEST-----
MIIFfzCCA2cCAQAwgbIxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJXQTEQMA4GA1UE
BxMHU2VhdHRsZTERMA8GA1UEChMITW9kdXNib3gxFDASBgNVBAsTC0VuZ2luZWVy
aW5nMR0wGwYDVQQDExRodWIuZGV2Lm1vZHVzYm94LmNvbTE8MDoGCSqGSIb3DQEJ
AQwtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3guY29t
MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAvzhLqKNx0n8dA95ohLNi
9ujpVwL109A3yZs5spXxc5ddPzhKiqWg9EAeYW0Bs+nPQ4jU/bqXNiO5ah+NDevh
nuoPgIkcZTzaI8wvwqblSeEPE66rnI15OFKKew0uWU7KugMLDnWZkm7WWPCvhGKh
ovmgN/9nI7aR4Ooh1TvUudqQ9w2f3TzP6IY5WNBgS6N+2peQ0GU+I7mJQfAOr/sS
TMNIV70c7iZf+6aZlekwLb3JLKGe6qO5fKw20U7UyPmeigrwzBc+5d8iBxCvjgWb
9ZWFvr+4C/GT042TtJE+34nmMEm3wLD34JDzYOthXT0pZQgkqHk0gD9Qi+rrvzGn
XwXIMvFBTbILfdVuWPcQ984h2JzKEkgdyC0ERlnmB1oNCKRPdobXvNv1sm3/22GJ
cRJ6ONzKR6IHpdXkVYjErUwfjs5WsNXVf+iQjaq+NdQgciiMYxr1fzZ1/i/Ba8H+
2+CbSPwY/rEAgXQBjpHZvgVZ8mwHAN13T0cfjLvq7zVlgBNFyreOQQ0D/4JV3Fsz
8XFt9DtGSZzDzQ5kauzoTN8IOBZToiVXYOzTTA4B5Awp3euyPwk0mDg+u1r0xqB0
fSeamv3h/lmDlBmzwdHEVCO7GOp15G7Y2NVvaQrQNey0S3N3gdr7da79Zv4Th7wT
J7l5mgX2YS/f/6j/U/Hi3ukCAwEAAaCBhjCBgwYJKoZIhvcNAQkOMXYwdDByBgNV
HREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22CFWh1YjIuZGV2Lm1vZHVzYm94
LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3gu
Y29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBDQUAA4ICAQBpYECe/uPNdtlgo3Io
aWUGikcSbh5OF9atGZHhsmZfjSmtYLWFWOrvYIhxLrnQmTLQCodnoLL3E/v44aNH
RtLhiltpFTEGL32JltcIkldFqnAVoQlDIllFqnp4ePsG+odUO2Ftjxd5H8Cw/HM7
DVgaMCZNklbMfckZkbziwVFExMHIcj1qp/2a7OkOA+y+aHxlQXVrRt/mvpmfBkWD
HSc3toiL/1HSmZI4YMqd2sh1Q6R+FiQrOvELzuwSV2S4A1ahW1jeB4/G+fTRZK5T
7xxpdUDl8AafzTRFji7grqQs8hUTobyCJozZW2za3mgZ2u0lRmNjVD89nB6D2/Va
/ScB6uNhPWLR2st+EEovCV7oIwQvckHnI9juAFml2g1G6U9xVNpOQgOMN9R6MVPZ
wK5D6+T3OPL5WD/BrOqPJBS0eIOE7sx6Pm39QZLWHjYX4GfI2JkkN6FbFyXlamwr
4huoM4KxPV5q72CPRzTr7ZSvUlzNOHF0HHu2SNwZ/uHpoHNDph0XVNGF4UIuvEdR
9vinWCEXlxuqOzkrgwqtNgpml10XY0YqhEbKl9LF83dXk1OfGw0E6vDB03Keos2c
H8HYeIXvvVpLFX3V6pwHVViyip0ntYHtP3e4jN5RnIbWBhPJrgd1/0pxN6KL/GpZ
a7mcOIcoGdtc7dc+u82647pj1g==
-----END CERTIFICATE REQUEST-----
