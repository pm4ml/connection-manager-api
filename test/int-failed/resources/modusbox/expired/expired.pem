-----BEGIN CERTIFICATE-----
MIIGgTCCBGmgAwIBAgIUE7xuaAKzL2bIFht2Rb3qDrOXs9UwDQYJKoZIhvcNAQEL
BQAwVTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQHEwdTZWF0dGxl
MREwDwYDVQQKEwhNb2R1c2JveDEUMBIGA1UECxMLRW5naW5lZXJpbmcwHhcNMTkw
NjA2MDEzNzAwWhcNMTkwNjA2MDIzNzAwWjCBsjELMAkGA1UEBhMCVVMxCzAJBgNV
BAgTAldBMRAwDgYDVQQHEwdTZWF0dGxlMREwDwYDVQQKEwhNb2R1c2JveDEUMBIG
A1UECxMLRW5naW5lZXJpbmcxHTAbBgNVBAMTFGh1Yi5kZXYubW9kdXNib3guY29t
MTwwOgYJKoZIhvcNAQkBDC1jb25uZWN0aW9uLW1hbmFnZXItYWRtaW5AaHViLmRl
di5tb2R1c2JveC5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQC/
OEuoo3HSfx0D3miEs2L26OlXAvXT0DfJmzmylfFzl10/OEqKpaD0QB5hbQGz6c9D
iNT9upc2I7lqH40N6+Ge6g+AiRxlPNojzC/CpuVJ4Q8TrqucjXk4Uop7DS5ZTsq6
AwsOdZmSbtZY8K+EYqGi+aA3/2cjtpHg6iHVO9S52pD3DZ/dPM/ohjlY0GBLo37a
l5DQZT4juYlB8A6v+xJMw0hXvRzuJl/7ppmV6TAtvcksoZ7qo7l8rDbRTtTI+Z6K
CvDMFz7l3yIHEK+OBZv1lYW+v7gL8ZPTjZO0kT7fieYwSbfAsPfgkPNg62FdPSll
CCSoeTSAP1CL6uu/MadfBcgy8UFNsgt91W5Y9xD3ziHYnMoSSB3ILQRGWeYHWg0I
pE92hte82/Wybf/bYYlxEno43MpHogel1eRViMStTB+Ozlaw1dV/6JCNqr411CBy
KIxjGvV/NnX+L8Frwf7b4JtI/Bj+sQCBdAGOkdm+BVnybAcA3XdPRx+Mu+rvNWWA
E0XKt45BDQP/glXcWzPxcW30O0ZJnMPNDmRq7OhM3wg4FlOiJVdg7NNMDgHkDCnd
67I/CTSYOD67WvTGoHR9J5qa/eH+WYOUGbPB0cRUI7sY6nXkbtjY1W9pCtA17LRL
c3eB2vt1rv1m/hOHvBMnuXmaBfZhL9//qP9T8eLe6QIDAQABo4HqMIHnMA4GA1Ud
DwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0G
A1UdDgQWBBSOmtQooHntMxpU/hvBb/HDfd2DoDAfBgNVHSMEGDAWgBRz+MYvsb3W
ZPU8obL/2gODY9F/yjByBgNVHREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22C
FWh1YjIuZGV2Lm1vZHVzYm94LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWlu
QGh1Yi5kZXYubW9kdXNib3guY29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBCwUA
A4ICAQB1v8iHu665JkBfJYwyGAu9hDjK74aTbCC6MZbiK18wMn58YtztZpMWjldE
qWbKBziGF0qY0S0Y95b+QbsGfWXPbjA+BFWbIFyO6gqjVqWuqG9RPVuAcwmAP2HW
xmUXgMKpJ8pD9G+nNY8rvAPIWXRD16og7z9y+g+NoxYlQKg2MlZhQupf/TNmEvlI
xJVcxQQpYHBtNh50JDfozK1uvvp70XpCB6SblxckfA4THQs5GUsphE9ylIZtjisO
mBJojx5AMMJgUqyyy9DO7AR+Ebv315OvqPl/ect/sWgqF1qAICnQH6n/PRnqrqr2
30fzpllFVeqw9W6Au5fHjWqcWPBjzFT9h3uOEmN660dvEZoQsmO9z78qJ+NqOD3+
mlUwTLbf51s6mz+Mr7juECl3KV5nhT8ROPkNfL9WdVz5zh6KJUWColmHyN6WG6gz
UJzeaVxHKv/W3YZhHZay5rEzJXZyz5CotBOXju5Vz9e5tOH/kGnorFyU1xnhChlu
6vfMWc3kcs/+/WY/F4bvz1GxqeDMU4CklTB6dChernjR91RjuNd1w2CSwVt/oINS
Y1sdUAqQkzo1HvWC2SAHpbzP1FEp3Gro+hT6Xf9epfgTzyCQK3yyem73ylFdxAXx
wOWXRRssg3ten9Z0op7tKLqwpNGHXa38wlY42BfnSroxS+49xg==
-----END CERTIFICATE-----
