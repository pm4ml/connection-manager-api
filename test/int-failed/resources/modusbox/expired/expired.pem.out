Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            13:bc:6e:68:02:b3:2f:66:c8:16:1b:76:45:bd:ea:0e:b3:97:b3:d5
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering
        Validity
            Not Before: Jun  6 01:37:00 2019 GMT
            Not After : Jun  6 02:37:00 2019 GMT
        Subject: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering, CN=hub.dev.modusbox.com/emailAddress=connection-manager-admin@hub.dev.modusbox.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:bf:38:4b:a8:a3:71:d2:7f:1d:03:de:68:84:b3:
                    62:f6:e8:e9:57:02:f5:d3:d0:37:c9:9b:39:b2:95:
                    f1:73:97:5d:3f:38:4a:8a:a5:a0:f4:40:1e:61:6d:
                    01:b3:e9:cf:43:88:d4:fd:ba:97:36:23:b9:6a:1f:
                    8d:0d:eb:e1:9e:ea:0f:80:89:1c:65:3c:da:23:cc:
                    2f:c2:a6:e5:49:e1:0f:13:ae:ab:9c:8d:79:38:52:
                    8a:7b:0d:2e:59:4e:ca:ba:03:0b:0e:75:99:92:6e:
                    d6:58:f0:af:84:62:a1:a2:f9:a0:37:ff:67:23:b6:
                    91:e0:ea:21:d5:3b:d4:b9:da:90:f7:0d:9f:dd:3c:
                    cf:e8:86:39:58:d0:60:4b:a3:7e:da:97:90:d0:65:
                    3e:23:b9:89:41:f0:0e:af:fb:12:4c:c3:48:57:bd:
                    1c:ee:26:5f:fb:a6:99:95:e9:30:2d:bd:c9:2c:a1:
                    9e:ea:a3:b9:7c:ac:36:d1:4e:d4:c8:f9:9e:8a:0a:
                    f0:cc:17:3e:e5:df:22:07:10:af:8e:05:9b:f5:95:
                    85:be:bf:b8:0b:f1:93:d3:8d:93:b4:91:3e:df:89:
                    e6:30:49:b7:c0:b0:f7:e0:90:f3:60:eb:61:5d:3d:
                    29:65:08:24:a8:79:34:80:3f:50:8b:ea:eb:bf:31:
                    a7:5f:05:c8:32:f1:41:4d:b2:0b:7d:d5:6e:58:f7:
                    10:f7:ce:21:d8:9c:ca:12:48:1d:c8:2d:04:46:59:
                    e6:07:5a:0d:08:a4:4f:76:86:d7:bc:db:f5:b2:6d:
                    ff:db:61:89:71:12:7a:38:dc:ca:47:a2:07:a5:d5:
                    e4:55:88:c4:ad:4c:1f:8e:ce:56:b0:d5:d5:7f:e8:
                    90:8d:aa:be:35:d4:20:72:28:8c:63:1a:f5:7f:36:
                    75:fe:2f:c1:6b:c1:fe:db:e0:9b:48:fc:18:fe:b1:
                    00:81:74:01:8e:91:d9:be:05:59:f2:6c:07:00:dd:
                    77:4f:47:1f:8c:bb:ea:ef:35:65:80:13:45:ca:b7:
                    8e:41:0d:03:ff:82:55:dc:5b:33:f1:71:6d:f4:3b:
                    46:49:9c:c3:cd:0e:64:6a:ec:e8:4c:df:08:38:16:
                    53:a2:25:57:60:ec:d3:4c:0e:01:e4:0c:29:dd:eb:
                    b2:3f:09:34:98:38:3e:bb:5a:f4:c6:a0:74:7d:27:
                    9a:9a:fd:e1:fe:59:83:94:19:b3:c1:d1:c4:54:23:
                    bb:18:ea:75:e4:6e:d8:d8:d5:6f:69:0a:d0:35:ec:
                    b4:4b:73:77:81:da:fb:75:ae:fd:66:fe:13:87:bc:
                    13:27:b9:79:9a:05:f6:61:2f:df:ff:a8:ff:53:f1:
                    e2:de:e9
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Client Authentication
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Subject Key Identifier: 
                8E:9A:D4:28:A0:79:ED:33:1A:54:FE:1B:C1:6F:F1:C3:7D:DD:83:A0
            X509v3 Authority Key Identifier: 
                keyid:73:F8:C6:2F:B1:BD:D6:64:F5:3C:A1:B2:FF:DA:03:83:63:D1:7F:CA

            X509v3 Subject Alternative Name: 
                DNS:hub1.dev.modusbox.com, DNS:hub2.dev.modusbox.com, email:connection-manager-admin@hub.dev.modusbox.com, IP Address:163.10.5.24, IP Address:163.10.5.21
    Signature Algorithm: sha256WithRSAEncryption
         75:bf:c8:87:bb:ae:b9:26:40:5f:25:8c:32:18:0b:bd:84:38:
         ca:ef:86:93:6c:20:ba:31:96:e2:2b:5f:30:32:7e:7c:62:dc:
         ed:66:93:16:8e:57:44:a9:66:ca:07:38:86:17:4a:98:d1:2d:
         18:f7:96:fe:41:bb:06:7d:65:cf:6e:30:3e:04:55:9b:20:5c:
         8e:ea:0a:a3:56:a5:ae:a8:6f:51:3d:5b:80:73:09:80:3f:61:
         d6:c6:65:17:80:c2:a9:27:ca:43:f4:6f:a7:35:8f:2b:bc:03:
         c8:59:74:43:d7:aa:20:ef:3f:72:fa:0f:8d:a3:16:25:40:a8:
         36:32:56:61:42:ea:5f:fd:33:66:12:f9:48:c4:95:5c:c5:04:
         29:60:70:6d:36:1e:74:24:37:e8:cc:ad:6e:be:fa:7b:d1:7a:
         42:07:a4:9b:97:17:24:7c:0e:13:1d:0b:39:19:4b:29:84:4f:
         72:94:86:6d:8e:2b:0e:98:12:68:8f:1e:40:30:c2:60:52:ac:
         b2:cb:d0:ce:ec:04:7e:11:bb:f7:d7:93:af:a8:f9:7f:79:cb:
         7f:b1:68:2a:17:5a:80:20:29:d0:1f:a9:ff:3d:19:ea:ae:aa:
         f6:df:47:f3:a6:59:45:55:ea:b0:f5:6e:80:bb:97:c7:8d:6a:
         9c:58:f0:63:cc:54:fd:87:7b:8e:12:63:7a:eb:47:6f:11:9a:
         10:b2:63:bd:cf:bf:2a:27:e3:6a:38:3d:fe:9a:55:30:4c:b6:
         df:e7:5b:3a:9b:3f:8c:af:b8:ee:10:29:77:29:5e:67:85:3f:
         11:38:f9:0d:7c:bf:56:75:5c:f9:ce:1e:8a:25:45:82:a2:59:
         87:c8:de:96:1b:a8:33:50:9c:de:69:5c:47:2a:ff:d6:dd:86:
         61:1d:96:b2:e6:b1:33:25:76:72:cf:90:a8:b4:13:97:8e:ee:
         55:cf:d7:b9:b4:e1:ff:90:69:e8:ac:5c:94:d7:19:e1:0a:19:
         6e:ea:f7:cc:59:cd:e4:72:cf:fe:fd:66:3f:17:86:ef:cf:51:
         b1:a9:e0:cc:53:80:a4:95:30:7a:74:28:5e:ae:78:d1:f7:54:
         63:b8:d7:75:c3:60:92:c1:5b:7f:a0:83:52:63:5b:1d:50:0a:
         90:93:3a:35:1e:f5:82:d9:20:07:a5:bc:cf:d4:51:29:dc:6a:
         e8:fa:14:fa:5d:ff:5e:a5:f8:13:cf:20:90:2b:7c:b2:7a:6e:
         f7:ca:51:5d:c4:05:f1:c0:e5:97:45:1b:2c:83:7b:5e:9f:d6:
         74:a2:9e:ed:28:ba:b0:a4:d1:87:5d:ad:fc:c2:56:38:d8:17:
         e7:4a:ba:31:4b:ee:3d:c6
